### ISSUE TITLE
Stopped working on Ventura 13.4.1

### ISSUE THREAD

**Issue by hupmake** [kirtan-shah/nowplaying-cli#10]
I keep my Mac pretty up-to-date and this has been working fine for awhile, but I upgraded to 13.4.1 this morning and it stopped working. Everything just returns null.

**Comment by my1e5** [comment]
I'm on 13.4 and I also got this problem.. see https://github.com/kirtan-shah/nowplaying-cli/issues/9

But for some reason, it seems to work again. I haven't fully diagnosed what has changed. But I found that, for example, if Apple Music is closed I get (null), but if I open Apple Music it works. I don't know, sorry this isn't a full solution. Just sharing my experience.

**Comment by hupmake** [comment]
Yeah, I just noticed I have it working again, too. That's really weird. 🤷‍♂️
